cmd/config/cluster-density-v2/deployment-client.yml

```diff
@@ -34,15 +34,18 @@ spec:
       containers:
       - name: client-app
         image: quay.io/cloud-bulldozer/curl:latest
-        command: ["sleep", "inf"]
+        command:
+            - "/bin/sh"
+            - "-c"
+            - "while true; do echo $(date +'%Y-%m-%d %H:%M:%S'): INFO curl on ${SERVICE_ENDPOINT} and ${ROUTE_ENDPOINT}; curl --fail -sS ${SERVICE_ENDPOINT} -o /dev/null && curl --fail -sSk ${ROUTE_ENDPOINT} -o /dev/null; sleep 120; done"
```

Comment:
Using commands is not the same as probes, as they don't generate load in the kubelet/cri-o side

Reply:
Im not sure how the pod will behave when a command like this fails, it will probably continue its execution regarldes if there's an error or not.

Reply:
> they don't generate load in the kubelet/cri-o side
True, this looping curl is simply to generate a live ingress traffic in the cluster. If you suggest to keep some active probes, can we add a probe for the sake of kubelet/cri-o may a probe that always gets true ? something that is not overwhelming router.

Reply:
Or I can do a conditional readiness probe, that would curl routes once and be simpler later that runs every 10s. 
```sh
            - "if [ ! -f /tmp/ready.log ]; then curl --fail -sS ${SERVICE_ENDPOINT} -o /dev/null && \ 
               curl --fail -sSk ${ROUTE_ENDPOINT} -o /dev/null && touch /tmp/ready.log; \ 
               else curl --fail -sS ${SERVICE_ENDPOINT} -o /dev/null; fi" 
```

Reply:
This command is fine as a readiness probe, where we just want to see if we are ready to serve the traffic in the beginning but for the rest of the period while do a 
```
            - "while true; do echo curl on ${ROUTE_ENDPOINT}; curl --fail -sSk ${ROUTE_ENDPOINT} -o /dev/null; sleep 120; done"
```
which basically is to see, if my app is still up and running, liveness probe would be the best fit instead of a command.

Reply:
How about setting `failureThreshold` high enough to a value that actually can be considered as a failure?

Reply:
Agree we can do, but additional probes would increase the load on kubelet. Anything is fine as long as we are not changing the load. As I said our intention of this command is only to generate some traffic from the client pod against the route endpoint without any load.